UniExtract.au3

```diff
@@ -692,21 +694,23 @@ Func ReadPrefs()
 		; user settings are stored in %appdata% due to permission issues
 		If CanAccess($globalIni) Then
 			Cout("Using global settings")
-			Global $settingsdir = @ScriptDir
+			$settingsdir = @ScriptDir
 		Else
 			Cout("Cannot write to " & $globalIni & ", using %appdata%")
 			FileCopy($globalIni, $userIni, 8)
 		EndIf
 	EndIf
 
 	; Setup paths
-	Global $prefs = $settingsdir & "\UniExtract.ini"
-	Global $batchQueue = $settingsdir & "\batch.queue"
-	Global $logdir = $settingsdir & "\log\"
-	Global $userDefDir = $settingsdir & "\def\"
-	Global $aDefDirs[] = [$userDefDir, $defdir]
-	Global $fileScanLogFile = $logdir & "filescan.txt"
-	Global Const $sPasswordFile = $settingsdir & "\passwords.txt"
+	$prefs = $settingsdir & "\UniExtract.ini"
+	$batchQueue = $settingsdir & "\batch.queue"
+	$logdir = $settingsdir & "\log\"
+	$userDefDir = $settingsdir & "\def\"
+	ReDim $aDefDirs[2]
+	$aDefDirs[0] = $userDefDir
+	$aDefDirs[1] = $defdir
```

Comment:
This has the same affect as `Global $aDefDirs[] = [$userDefDir, $defDir]`, it just doesn't re-declare the global variable.